Sentry.xcodeproj/project.pbxproj

```diff
@@ -3756,48 +3762,54 @@
 		8ECC673625C23936000E2BF6 /* Transaction */ = {
 			isa = PBXGroup;
 			children = (
+				D8603DD7284F894C000E1227 /* SentryBaggage.h */,
```

Comment:
These changes originate from me sorting the list of files in the Xcode file explorer